software/map_downloader.py

```diff
@@ -0,0 +1,55 @@
+import requests
+import os
+
+"""
+Helper class to download a small hybrid Google map, with a marker
+at at a specified lat-long coordinate. The file is then saved to a
+specified {filename}.png
+
+Created 07 September 2016
+By Marcel van Workum
+
+Modified 07 September 2016
+By Marcel van Workum
+"""
+class MapDownloader:
+    API_ROOT = "https://maps.googleapis.com/maps/api/staticmap?"
+    API_ID = "google_maps_api_key"
+    API_SEPARATOR = "&"
+    SIZE = "size=400x400"
+    ZOOM = "zoom=16"
+    MAP_TYPE = "hybrid"
+    MARKER_TEMPLATE = "markers=color:red%7Clabel:R%7C"
+
+    def __init__(self, location, filename):
+        # initialize the location and center of map, setting a mark at the location
+        self.location = location
+        self.center = "center=" + location
```

Comment:
Should document the format of this location variable. Reading through it should me "lat,long", but should be commented somewhere